examples/Example/src/pagerAndDrawer/index.android.tsx

```diff
@@ -31,7 +37,9 @@ export default class Example extends Component {
         <View>
           <DrawerLayoutAndroid
             drawerWidth={200}
-            drawerPosition={DrawerLayoutAndroid.positions.Left}
+            // This is changed from DrawerLayout.positions.Left and DrawerLayout.positions.Right
+            // because RN uses literal `left` and `right` strings
+            drawerPosition="left"
```

Comment:
We should document it somewhere.

Reply:
Did you do it?

Reply:
No, I looked at RN docs and they use literal strings.

Actually, do we need this comment?

Reply:
I think it is not necessary.